src/pages/workspaces/workspace/tools/WorkflowView.js

```diff
@@ -309,13 +315,14 @@ const WorkflowView = _.flow(
   render() {
     const { isFreshData, savedConfig, launching, activeTab, variableSelected, modifiedConfig } = this.state
     const { namespace, name, workspace } = this.props
+    const { rootEntityType } = modifiedConfig
```

Comment:
I'm getting a "cannot read rootEntityType of undefined" error when I try to open the tools worfklowView on your branch. 
<img width="1792" alt="screen shot 2018-12-06 at 1 29 16 pm" src="https://user-images.githubusercontent.com/42386483/49604077-f2c26880-f95a-11e8-8db2-0f2e588a4a7c.png">

Reply:
Thanks for looking. I rebased my branch before opening the PR and sorta rushed through a merge conflict. I did test it, though, and it seems to work for me. Is this a workspace you can share with me? (breilly.firecloud@gmail.com)

Reply:
I'm using the badm test user on the son-of-jimmy workspace

Reply:
Should be fixed now. Something related to the initial render which is why I didn't catch it (even a reload works fine 🤷‍♂️ ). I prefer the updated code anyway.